schemas/procedures.json

```diff
@@ -76,6 +82,100 @@
     },
     
     "$defs": {
+        "implant": {
+            "type": "object",
+            "properties": {
+                "date": {
+                    "type": "string",
+                    "format": "date",
+                    "title": "Date"
+                },
+                "experimenter_full_name": {
+                    "type": "string",
+                    "title": "Experimenter full name",
+                    "description": "First and last name of the experimenter."
+                },
+                "probes": {
+                    "type": "array",
+                    "items": {"$ref": "#/$defs/probe"},
+                    "uniqueItems": true,
+                    "title": "Probes"
+                }
+            },
+            "required": ["date", "experimenter_full_name","probes"]
+        },
+        "probes": {
```

Comment:
Given the ref on line 100, I think this should be "probe"

Reply:
oh, this is actually something else. Too many different types of probes. Let me find a different name for one of these.

Reply:
ah, that's right. Fixing it now.